puppet/modules/jenkins_job_builder/manifests/config.pp

```diff
@@ -6,46 +6,28 @@
   String $password,
   Integer[0] $jenkins_jobs_update_timeout = 600,
   String $command_arguments = 'foreman-infra-jenkins-job-update',
+  String $git_project_name = 'foreman-infra',
+  String $git_repo = 'https://github.com/theforeman/foreman-infra.git',
```

Comment:
I'd prefer to do it right the first time and created https://github.com/theforeman/jenkins-jobs. It also includes the history for git log and git blame.

Reply:
I appreciate you making the repository. As the person making the change I did prefer to get the mechanism to work and then orchestrate the moving of the jobs given there are open PRs and such still to reduce the amount of potential churn on all fronts. At this point, I'll just update towards the ultimate goal and hope it works out.